apps/src/code-studio/initializeBlockPreview.js

```diff
@@ -0,0 +1,36 @@
+function update(blockSpace, container, editor) {
+  try {
+    // populate with new xml; because we do this every time the editor
+    // updates, there is the potential for many different kinds of
+    // errors here as we will end up updating as the levelbuilder is
+    // typing out the new content. Because of that and the fact that
+    // this is simply an informative but not functional view, we simply
+    // catch and ignore all errors.
+    var xml = $.parseXML(editor.getValue()).childNodes[0];
+    blockSpace.clear();
+    Blockly.Xml.domToBlockSpace(blockSpace, xml);
+  } catch (e) {
```

Comment:
Seeing a number of caught exceptions in `blockSpace.clear();` which I wasn't expecting.